website/common/script/content/constants/index.js

```diff
@@ -2,11 +2,12 @@ import moment from 'moment';
 
 export const CURRENT_SEASON = moment().isBefore('2020-08-02') ? 'summer' : '_NONE_';
```

Comment:
Given we haven't touched this since three years ago, I'm guessing this is unused and can be removed

Reply:
removed